task/manager.go

```diff
@@ -24,12 +24,14 @@ import (
 	"github.com/konveyor/tackle2-hub/model"
 	"github.com/konveyor/tackle2-hub/reflect"
 	"github.com/konveyor/tackle2-hub/settings"
-	"gopkg.in/yaml.v2"
 	"gorm.io/gorm"
 	core "k8s.io/api/core/v1"
 	k8serr "k8s.io/apimachinery/pkg/api/errors"
 	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	k8r "k8s.io/apimachinery/pkg/runtime"
+	k8j "k8s.io/apimachinery/pkg/runtime/serializer/json"
+	k8y "k8s.io/apimachinery/pkg/runtime/serializer/yaml"
```

Comment:
**hint**: using this ^ because it's a json encoder that will render yaml.  This is helpful because it will respect the json tags (especially for omitempty).